components/Header/Header.js

```diff
@@ -1,28 +1,373 @@
+            >
+              Disclaimer
+            </span>
+            &nbsp;and&nbsp;
+            <span
+              tabIndex="0"
+              className={classes.clickableSubheaderText}
+              onClick={this.openUserPrivacyStatement}
+              onKeyDown={this.openUserPrivacyStatement}
+              role="link"
+            >
+              User Privacy Statement
+            </span>
+            &nbsp;before using our catalog.
+          </Grid>
+        </Grid>
+
+        <Dialog
+          onClose={this.closeDisclaimerDialog}
+          open={isDisclaimerDialogOpen}
+        >
+          <DialogTitle
+            onClose={this.closeDisclaimerDialog}
+            className={classes.dialogTitle}
+          >
+            AsylumConnect Disclaimer
+            <div
+              className={classes.closeDialogX}
+              onClick={this.closeDisclaimerDialog}
+              onKeyDown={this.closeDisclaimerDialog}
+              role="button"
+              tabIndex="0"
+            >
+              x
+            </div>
+          </DialogTitle>
+          <DialogContent>
+            The AsylumConnect team will do its best to confirm basic facts about
+            service providers listed on this website. However, we lack the
+            necessary resources to fully vet service providers and make no
+            representations regarding the viability or capabilities of any such
+            providers. Consequently, AsylumConnect assumes no responsibility for
+            the actions of providers listed on this website and asylum seekers
+            who contact any such providers do so at their own risk.
+          </DialogContent>
+          <DialogContent className={classes.dialogOkButtonContainer}>
+            <MuiThemeProvider theme={redTheme}>
+              <Fab
+                color="primary"
+                size="small"
+                style={{ width: '100px' }}
+                onClick={this.closeDisclaimerDialog}
+                variant="extended"
+              >
+                Ok
+              </Fab>
+            </MuiThemeProvider>
+          </DialogContent>
+        </Dialog>
+
+        <Dialog
+          onClose={this.closeUserPrivacyStatement}
+          open={isUserPrivacyStatementDialogOpen}
+        >
+          <DialogTitle
+            onClose={this.closeUserPrivacyStatement}
+            className={classes.dialogTitle}
+          >
+            AsylumConnect User Privacy Statement
+            <div
+              className={classes.closeDialogX}
+              onClick={this.closeUserPrivacyStatement}
+              onKeyDown={this.closeUserPrivacyStatement}
+              role="button"
+              tabIndex="0"
+            >
+              x
+            </div>
+          </DialogTitle>
+          <DialogContent>
+            The AsylumConnect catalog uses Google Analytics with&nbsp;
+            <a href="https://support.google.com/analytics/answer/2763052?hl=en">
```

Comment:
```suggestion
            <a href="https://support.google.com/analytics/answer/2763052">
```

Seems like we shouldn't be specifying what language the reader wants to read the google support document in. If they have their language preference set in the browser or their google support cookies, let's save them a click.